README.md

```diff
@@ -151,28 +149,71 @@ constants must be common to all clients and the server:
  3. `TIMEOUT` In ms. Normally 1500. See sections 6 and 7.
  4. `CLIENT_ID` Associates an ESP8266 with its server-side application. Must be
  unique to each client. May be any `\n` terminated Python string.
+ The client and server configuration is done using constructor arguments, 
+ therefore `local.py` is imported in the example files and can be implemented differently. 
 
 ## 3.1 Installation
 
 It is recommended to use the latest release build of firmware as such builds
 incorporate `uasyncio` as frozen bytecode. Daily builds do not. With a release
-build copy the above client files to the device. Edit `local.py` as described
-below and copy it to the device. Ensure the device has a stored WiFi connection
-and run te demo.
+build copy the above client files to the device. Edit `local.py` for each example
+as described below and copy it to the device. Ensure the device has a stored 
+WiFi connection and run the demo.
+To run the demo the file `local.py` for the corresponding example should be edited 
+for the server IP address.
+The demo supports up to four clients. Each client's `local.py` should be edited
+to give each client a unique client ID in range 1..4. Note that the ESP8266
+must have a stored network connection to access the server.
 
-Alternatively to maximise free RAM firmware can be built from source, freezing
+On the server ensure that `local.py` is on the path and run `s_app_cp.py`.
+
+Alternatively to maximise free RAM, firmware can be built from source, freezing
 `uasyncio`, `client.py` and `primitives.py` as bytecode.
 
 If a daily build is used it will be necessary to
 [cross compile](https://github.com/micropython/micropython/tree/master/mpy-cross)
 `client.py`
 
-To run the demo the file `local.py` should be edited for the server IP address.
-The demo supports up to four clients. Each client's `local.py` should be edited
-to give each client a unique client ID in range 1..4. Note that the ESP8266
-must have a stored network connection to access the server.
+To get the files onto your ESP8266 do NOT copy single files as this repository is
```

Comment:
rshell synchronization of the package.
Sadly the image file causes a little error message but rshell rsync has no exclude option yet.